### ISSUE TITLE
Version for ArcGIS 10.4

### ISSUE THREAD

**Issue by njackson01** [Esri/arcgis-osm-editor#116]
When do you plan have a version released for ArcGIS 10.4?

**Comment by slead** [comment]
argh. I was running ArcGIS 10.2 and got the message about this extension requiring 10.3, so I upgraded my ArcGIS. "While I'm here, why not upgrade to 10.4?" I figured. Now I know - this extension requires 10.3, exactly.

+1 to considering releasing a 10.4 version.

And for future releases, would it make sense to check for the _minimum_ version, rather than the _exact_ version? Thanks

**Comment by ThomasEmge** [comment]
Agreed. However 10.4 is a significant change that does require a re-compile of the toolset.

**Comment by njackson01** [comment]
So when do you think an update toolset will be posted for 10.4?

**Comment by eggwhites** [comment]
We are finalizing testing of our 10.4 builds, and hope to release the 10.4 version very soon.  I know "very soon" is not helpful - but we are targeting within the next couple of weeks.

**Comment by njackson01** [comment]
@eggwhites thanks for the update. look forward to the release.

**Comment by mboeringa** [comment]
> @eggwhites thanks for the update. look forward to the release.

Appears released now:
[ArcGIS Editor for OSM 10.4.x Desktop](http://www.arcgis.com/home/item.html?id=c18d3d0d5c62465db60f89225fdd2698)

**Comment by njackson01** [comment]
@mboeringa yeah I saw it last week. downloaded and it seems to be working just fine.

**Comment by nasriadam3** [comment]
is possible to use ARC GIS 10.4 to analyse satellite images?
